From 56a53ee1c11b778e79e2ad7ee7e309e9917fd13c Mon Sep 17 00:00:00 2001
From: Ross Lagerwall <ross.lagerwall@citrix.com>
Date: Wed, 11 Oct 2017 17:49:48 +0200
Subject: [PATCH] x86: fix XEN_DMOP_remote_shutdown return value

Return 0 to indicate success rather than whatever rc was previously set
to (-EINVAL).

Signed-off-by: Ross Lagerwall <ross.lagerwall@citrix.com>
Reviewed-by: Jan Beulich <jbeulich@suse.com>
diff --git a/xen/arch/x86/hvm/dm.c b/xen/arch/x86/hvm/dm.c
index e348f1a..9236afa 100644
--- a/xen/arch/x86/hvm/dm.c
+++ b/xen/arch/x86/hvm/dm.c
@@ -568,6 +568,7 @@ static int dm_op(const struct dmop_args *op_args)
             &op.u.remote_shutdown;
 
         domain_shutdown(d, data->reason);
+        rc = 0;
         break;
     }
 
